Generals/Code/GameEngine/Source/GameClient/MessageStream/LookAtXlat.cpp

```diff
@@ -430,8 +435,9 @@ GameMessageDisposition LookAtTranslator::translateGameMessage(const GameMessage
 						// TheSuperHackers @info calculate the length of the vector to obtain the movement speed before the vector is normalized
 						float vecLength = vec.length();
 						vec.normalize();
-						offset.x = TheGlobalData->m_horizontalScrollSpeedFactor * logicToFpsRatio * vecLength * vec.x * TheGlobalData->m_keyboardScrollFactor;
-						offset.y = TheGlobalData->m_verticalScrollSpeedFactor * logicToFpsRatio * vecLength * vec.y * TheGlobalData->m_keyboardScrollFactor;
+						// TheSuperHackers @tweak Rescale the scroll factor so the default scroll speed at 50 will result in the RMB scrolling being identical to the original 30FPS retail behavior
+						offset.x = TheGlobalData->m_horizontalScrollSpeedFactor * logicToFpsRatio * vecLength * vec.x * TheGlobalData->m_keyboardScrollFactor * SCROLL_MULTIPLIER;
+						offset.y = TheGlobalData->m_verticalScrollSpeedFactor * logicToFpsRatio * vecLength * vec.y * TheGlobalData->m_keyboardScrollFactor * SCROLL_MULTIPLIER;
```

Comment:
Perhaps write this as

`SCROLL_MULTIPLIER * TheGlobalData->m_keyboardScrollFactor`

to match 

`SCROLL_AMT * TheGlobalData->m_keyboardScrollFactor`